src/reformat_libyuv.c

```diff
@@ -80,6 +80,9 @@ unsigned int avifLibYUVVersion(void)
 // These defines are used to create a NULL reference to libyuv functions that
 // did not exist prior to a particular version of libyuv.
 // Versions prior to 1755 are considered too old and not used (see CMakeLists.txt).
+#if LIBYUV_VERSION < 1903
```

Comment:
Optional: RAWToJ444 was added in version 1902, but the Neon optimization was added later without bumping the version. And then RAWToI444 was added in version 1903. So it is fine to test 1903 here. I suggest citing the relevant CLs:

https://chromium-review.googlesource.com/c/libyuv/libyuv/+/6207845
https://chromium-review.googlesource.com/c/libyuv/libyuv/+/6220890
https://chromium-review.googlesource.com/c/libyuv/libyuv/+/6223658

See the comment at lines 96-97 as an example.

Reply:
> See the comment at lines 96-97 as an example.

I'll add them, but they feel redundant given the `LIBYUV_VERSION` check. That and other examples are undocumented.